xep-0448.xml

```diff
@@ -146,6 +151,12 @@ Note to editor: Remove xep-sfs entity declared below and change all references f
       <td>Key: 256, IV: 96</td>
       <td>GCM/NoPadding</td>
     </tr>
+    <tr>
+      <td>urn:xmpp:ciphers:aes-256-cbc-pkcs7:0</td>
```

Comment:
Question: Do we need versioning for this namespace (`:0`)? :)

Reply:
I sticked with it when copying from [XEP-0391](https://xmpp.org/extensions/xep-0391.html#ciphers). We probably want to factor this out of both XEPs before they go stable and then should revisit the choice of having a version (it might make sense if we wanted to change things like IV size, but that's not even possible for many ciphers) - it certainly doesn't hurt for now.

Reply:
Ah okay, I wasn't aware of that it's already used elsewhere :) Then this is probably totally fine :)